vertical-pod-autoscaler/RELEASE.md

```diff
@@ -0,0 +1,159 @@
+# VPA Release Instructions
+
+These are instructions for releasing VPA. We aim to release a new VPA minor version after each minor Kubernetes release.
+We release patch versions as needed.
+
+Before doing the release for the first time check if you have all the necessary permissions.
+
+There are the following steps of the release process:
+
+1. [ ] Open issue to track the release.
+2. [ ] Update VPA version const.
+3. [ ] Build and stage images.
+4. [ ] Test the release.
+5. [ ] Promote image.
+6. [ ] Finalize release.
+
+## Open issue to track the release
+
+We use the issue to communicate what is state of the release.
+
+## Update VPA version const
+
+### New minor release
+
+1. [ ] Wait for all VPA changes that will be in the release to merge.
+2. [ ] Wait for [the end to end tests](https://k8s-testgrid.appspot.com/sig-autoscaling-vpa) to run with all VPA changes
+    included.
+3. [ ] Make sure the end to end VPA tests are green.
+4. [ ] Change the version in
+    [common/version-go](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/common/version.go)
+    to `0.${next-minor}.0`,
+5. [ ] Commit and merge the change,
+6. [ ] Go to the merged change,
+7. [ ] [Create a new branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) named `vpa-release-0.${next-minor}` from the
+    merged change.
+
+### New patch release
+
+1.  [ ] Bump the patch version number in VerticalPodAutoscalerVersion constant in
```

Comment:
Bit nit-picking but I hope that steps 1-3 from new minor release also apply here, right?

Reply:
Yes, I moved those steps a level up